base_export_async/tests/test_base_export_async.py

```diff
@@ -64,22 +64,26 @@ def test_export_csv(self):
         params = json.loads(data_csv.get("data"))
         mails = self.env["mail.mail"].search([])
         attachments = self.env["ir.attachment"].search([])
-        self.delay_export_obj.export(params)
+        self.delay_export_obj.export(
+            params, self.env.ref("base_export_async.delay_export_mail_template"), True
+        )
         new_mail = self.env["mail.mail"].search([]) - mails
-        new_attachment = self.env["ir.attachment"].search([]) - attachments
+        new_attachments = self.env["ir.attachment"].search([]) - attachments
```

Comment:
there should be only one attachment, isn't it?

Reply:
There will be 2 attachments, one attached to the mail and one for the link in the mail.